lwt/dune

```diff
@@ -1,5 +1,36 @@
+(* -*- tuareg -*- *)
+(* This was inspired by `conduit-lwt-unix`'s dune file *)
+
+let v ~ssl ~tls () =
+  let ssl, ssl_d =
+    if ssl then "ssl_io_real", "lwt_ssl "
+    else "ssl_io_dummy", ""
+  in
+  let tls, tls_d =
```

Comment:
It looks like in this one you've taken a pretty different approach from the one in #92 - can you explain why they're so different?  Sorry, I'm not super familiar with dune file conventions.

Reply:
I went with different approaches because there isn't an OCaml-TLS binding for Async yet. It makes things easier there because there's only one library to choose from.